cms/src/api/layer/content-types/layer/schema.json

```diff
@@ -40,10 +40,41 @@
     "interaction_config": {
       "type": "json"
     },
-    "metadata": {
-      "type": "component",
-      "repeatable": false,
-      "component": "documentation.metadata"
+    "source": {
+      "type": "string"
+    },
+    "source_url": {
+      "type": "string"
+    },
+    "description": {
+      "type": "text"
+    },
+    "name": {
```

Comment:
I'm not sure if it's because my database is older but I see two fields on Strapi now: `title` and `name`. Only one of them is needed.

On [Ángel's spreadsheet](https://docs.google.com/spreadsheets/d/1k0TfolRxGYKj09fGkzAzyXes9WVvkj53ShMqdTafkzo/edit#gid=1647775375), only the content of the “Name” column is relevant. The column “Layer (unofficial name)” won't be stored on Strapi.

Reply:
Ok, Ill delete the name field